### ISSUE TITLE
Trace counters aren't present in timeline

### ISSUE THREAD

**Issue by RomanArzumanyan** [google/perfetto#803]
Hello world,

I'm trying to monitor memory usage with trace counters. I'm following this sample:
https://github.com/google/perfetto/blob/0fbd3808d530f8c85568a4b8e9975756ecffa4bd/examples/sdk/example.cc#L87-L97

Application starts and works fine, tracing session starts and stop without any errors or exceptions.
But for some reason, no trace counters are presented in JSON file I obtain from `traceconv`.

Basically, the timeline I see is similar to the example: https://github.com/google/perfetto/blob/main/examples/sdk/example.png

There are plenty of trace events, but no trace counters. 
Any help is much appreciated!

**Comment by aMayzner** [comment]
Hi!

If you open the trace in the UI before using traceconv (.pb or
.perfetto_trace) can you see trace counters? Our JSON support is best
effort so it might be related to losing some data in the conversion.

**Comment by RomanArzumanyan** [comment]
Hi @aMayzner 

I've opened the binary trace file and it does solve the problem. Can see all my counters there.
Thanks a lot for the help!
